app/models/vmdb_table.rb

```diff
@@ -19,4 +48,21 @@ def my_metrics
   def self.display_name(number = 1)
     n_('Table', 'Tables', number)
   end
+
+  def sql_indexes
+    if has_attribute?(:all_indexes)
+      self["all_indexes"].split("|").map do |indx|
+        index_name, unique, *columns = indx.split(",")
+        ActiveRecord::ConnectionAdapters::IndexDefinition.new(name, index_name, unique == 't', columns.map(&:strip))
```

Comment:
Okay, good, you are doing a `columns.map(&:strip)` here.  This is important since sometimes there are two columns (or more) in an index, and that has a space that you aren't striping out as part of `all_indexes` SQL.

You might want to add a spec for this case, or at the very least, a comment (my comment example above references this).  This is SUPER important that this isn't incorrect.